src/test/java/core/enums/ContentType.java

```diff
@@ -0,0 +1,23 @@
+package core.enums;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public enum ContentType {
```

Comment:
there's the same `io.restassured.http.ContentType` rest assured enum

Reply:
yes, removed. it even  isn't used(